### PR TITLE
add bounce and acc

### PR DESCRIPTION
**Pull request by erikgaas**

### CODE REVIEW

**Comment by erikgaas** [comment]
👍

**Comment by erikgaas** [comment]
🚀 🙏

**Comment by erikgaas** [comment]
👎

**Comment by erikgaas** [comment]
meow @ericdudley

File: index.js [hunk 1]
```diff
@@ -16,9 +25,9 @@ function setup() {
     dots.push(new Dot());
     dots[i].pos.x = Math.random() * width;
     dots[i].pos.y = Math.random() * height;
-    dots[i].rad = 15 + Math.random() * 10;
-    dots[i].vel.x = Math.random() * 5;
-    dots[i].vel.y = Math.random() * 5;
+    dots[i].rad = 15;
+    dots[i].vel.x = (Math.random() - 0.5);
```
Comment (ericdudley) [inline index.js:29]: Whooooa, I've never thought to do this. I've always done `(Math.random() > 0.5 ? 1 : -1) * expression`

File: index.js [hunk 2]
```diff
@@ -6,6 +6,15 @@ function windowResized() {
   resizeCanvas(rect.width, rect.height);
 }
 
+function dotStep(dot, delta) {
```
Comment (ericdudley) [inline index.js:9]: You should put this in `Dot.update`

File: index.js [hunk 3]
```diff
@@ -28,13 +37,19 @@ function draw() {
...
     dots[i].update();
     dots[i].render();
+    dotStep(dots[i], 0.001);
 
     for (let j = 0; j < dots.length; j++) {
       const other = dots[j];
       const other_modpos = other.modpos;
       if (dots[i].modpos.dist(other.modpos) < 100 && dots[i].infected) {
         if (Math.random() > 0.5) {
+          if (dots[j].infected  == false) {
```
Comment (ericdudley) [inline index.js:47]: nit `!dots[j].infected`

**Review by ericdudley — APPROVED** [review]

**Pull request merged**